server/controllers/dbcontroller.js

```diff
@@ -55,7 +55,7 @@ const authenticateUser = async (user, password) => {
 
 const loadAll = async () => {
 	const result = await db.any(
-		'SELECT doggos.doggoid, doggoname, description, imageurl, username, SUM(likes.likecount) AS likestotal FROM doggos INNER JOIN likes ON doggos.doggoid = likes.doggoid GROUP BY doggos.doggoid, likes.likecount'
+		'SELECT doggos.doggoid, doggoname, imageurl, description, username, SUM(likes.likecount) AS likestotal FROM doggos LEFT JOIN likes ON doggos.doggoid = likes.doggoid GROUP BY doggos.doggoid, likes.likecount'
```

Comment:
So why are you doing a left join instead of inner?

Reply:
So inner join will not show dogs with 0 likes (NULL), while left join will